pyiron_base/jobs/master/serial.py

```diff
@@ -176,41 +173,6 @@ def ref_job(self):
     def ref_job(self, job):
         self.append(job)
 
-    @property
-    def input(self):
-        """
-        Get the input of the start job - the first job of the series.
-
-        Returns:
-            GenericParameters: input of the start job
-        """
-        if self.start_job is not None:
-            return self._start_job.input
-        else:
-            return None
-
-    @input.setter
-    def input(self, value):
-        """
-        Set the input of the start job - the first job of the series.
-
-        Args:
-            value (GenericParameters): input of the start job
-        """
-        if self.start_job is not None:
-            self._start_job.input = value
-        else:
-            raise ValueError(
-                "Input can only be set after a start job has been assinged."
-            )
```

Comment:
Here the functionality changes - rather than accessing the input of the start_job - the template other jobs are generated from - we now access the input of the master, still I think this is more clear this way.

Reply:
TBH I'm a little confused how it could be any other way. Isn't it possible/desirable/*necessary* that the serial master has different input from its reference job??

Reply:
> TBH I'm a little confused how it could be any other way. Isn't it possible/desirable/_necessary_ that the serial master has different input from its reference job??

I guess the initial idea was that you could still access the start job from the serial master, so when you get the input you get the input of the start job and correspondingly when you get the output of the serial master you get the output of the last calculation. But I agree that separating both is more clean, so that is the way I designed this pull request.